### PR TITLE
docs: fix interval type in the example

### PR DESCRIPTION
**Pull request by fabien0102**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] ~~Tests for the changes have been added (for bug fixes / features)~~
- [ ] ~~Docs have been added / updated (for bug fixes / features)~~

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

`window.setInterval` returns a `number`, but the example is using `any`.

Issue Number: N/A

## What is the new behavior?

The new behavior is to use the correct type for `setInterval`

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

~TypeScript can be confused between nodeJS and browser `setInterval`, using `window.setInterval` instead of the implicit `setInterval` remove this issue.~

### CODE REVIEW

**Review by jessicajaniuk — APPROVED** [review]
LGTM 🍪 

Thank you!

File: aio/content/examples/dynamic-component-loader/src/app/ad-banner.component.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export class AdBannerComponent implements OnInit, OnDestroy {
   }
 
   ngOnDestroy() {
-    clearInterval(this.interval);
+    window.clearInterval(this.interval);
```
Comment (AndrewKushnir) [inline aio/content/examples/dynamic-component-loader/src/app/ad-banner.component.ts:34]: @fabien0102 thanks for creating this PR. The change to have more precise type looks great, but I'm not sure if we should use `window` here. I've looked at other cases in docs and we seem to use `setInterval` and `setTimeout` without the `window` reference, so I'd propose keeping just the type change and revert the adding `window.`.

Comment (fabien0102) [reply aio/content/examples/dynamic-component-loader/src/app/ad-banner.component.ts]: Let's see if this is compiling without `window.` (I did have issue before with some conflicts with the `clearInterval` from nodejs global types)

Comment (fabien0102) [reply aio/content/examples/dynamic-component-loader/src/app/ad-banner.component.ts]: Works!

**Comment by AndrewKushnir** [comment]
@fabien0102 thanks for updating the PR, the changes look good 👍 I'm adding this PR to the merge queue.

Thanks for improving Angular docs!

**Review by AndrewKushnir — APPROVED** [review]

**Pull request closed without merging**